docs/baseband/vdif/index.rst

```diff
@@ -159,7 +153,7 @@ supported by Baseband, support can be added by implementing a custom header
 class.  If the EDV is supported, but the header deviates from the format
 found in the `VLBI.org EDV registry <http://www.vlbi.org/vdif/>`_, the
 best solution is to create a custom header class, then override the
-subclass selector in :class:`~baseband.vdif.header.VDIFHeader`.  Tutorials
```

Comment:
Here and elsewhere: why has the `:class:` been removed?

Reply:
They don't seem to do anything, and Sphinx can generate links to the classes without them.

Reply:
Still not sure there is a reason to remove it -- a sphinx style may well change to distinguish between types of links, and then we have removed the relevant information. (Though on the other hand sphinx should be able to tell from the link itself.)

Reply:
We currently do this inconsistently throughout both the docs and the API.  For standardization purposes, I prefer getting rid of them all, and putting them back in if Sphinx ever breaks.  This is to reduce the amount of redundant text, and because I keep forgetting which tags exist.

Reply:
My overall sense would be to not change if unsure. Possibly, it is better to just define the most use classes up front and use `|VDIFHeader|`. Anyway, not a big deal.